spark/core.py

```diff
@@ -512,4 +514,19 @@ def check_coords(self, func_name, *args, width_only=False):
     def arc_args(self, *args):
         return (args[0], args[1], args[2] / 2, 0, 2 * pi)
 
+    # Global namespace alias of random.random()
+    def random(self, *args):
+        argc = len(args)
+        if argc != 0:
+            raise ArgumentNumError("Random", 0, argc)
```

Comment:
Should be lowercase to match the function name
```suggestion
            raise ArgumentNumError("random", 0, argc)
```

Reply:
Resolved in latest commit